be/src/formats/parquet/column_reader.cpp

```diff
@@ -178,6 +255,17 @@ Status ColumnReader::create(const ColumnReaderOptions& opts, const ParquetField*
         std::unique_ptr<ListColumnReader> reader(new ListColumnReader(opts));
         RETURN_IF_ERROR(reader->init(field, std::move(child_reader)));
         *output = std::move(reader);
+    } else if (field->type.type == TYPE_MAP) {
+        std::unique_ptr<ColumnReader> key_reader;
+        // ParquetFiled Map -> Map<Struct<key,value>>
+        RETURN_IF_ERROR(
+                ColumnReader::create(opts, &(field->children[0].children[0]), col_type.children[0], &key_reader));
```

Comment:
do we need some `DCHECK` for verifying the type of children and the num of children, etc.